javascript/javascript1/week1/dog-age-calci.js

```diff
@@ -0,0 +1,18 @@
+let dogYearOfBirth = 2010;
+let dogYearFuture = 2022;
+let dogYear = dogYearFuture - dogYearOfBirth;
+
+var shouldShowResultInDogYears = 'True';
```

Comment:
- In this assigning `'True'` is a string, you can assign `true` as a boolean instead.
- Try to use always `const` and `let` not `var`